src/popup/components/Modals/ConfirmRawSign.vue

```diff
@@ -28,7 +28,7 @@
     </div>
 
     <DetailsItem
-      :label="$t('modals.confirm-transaction-sign.data-sign')"
+      :label="$t('modals.confirmTransactionSign.data-sign')"
```

Comment:
why part of the translation key is in camelCase and part in kebab-case?

Reply:
Guess what, someone did interpolation:
```
:label="$t('modals.confirm-transaction-sign')[key]"
```
For now I don't want to mess this up. But in the future we definitely need to refactor this.